.gitignore

```diff
@@ -10,3 +10,4 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+.idea/
```

Comment:
This is still here ;)

Reply:
Yeah I've added that in the `$HOME/.gitignore` to ignore the goland folder completely. I remove the actual folder from the repo that was commited previously.